app/views/pages/privacy_policy.html.erb

```diff
@@ -5,16 +5,19 @@
               class: 'govuk-back-link',
               data: {controller: 'back-link'} %>
 
-    <h1 class="govuk-heading-l">Privacy notice for the School Experience Service</h1>
+    <h1 class="govuk-heading-l">Privacy policy for the get school experience</h1>
```

Comment:
Should this line have service at the end? Reads a bit odd without it

Reply:
or maybe lose the "the"

Reply:
Yes, you're right. I'll remove the 'the'